llvm/utils/mlgo-utils/mlgo/corpus/combine_training_corpus.py

```diff
@@ -35,11 +36,21 @@ def parse_args_and_run():
     parser.add_argument(
         "--root_dir", type=str, help="The root dir of module paths to combine."
     )
+    parser.add_argument(
```

Comment:
can you add an issue and reference it here and below to factor in a common place stuff like this? otherwise LGTM.